packages/keystone/src/___internal-do-not-use-will-break-in-patch/node-api.ts

```diff
@@ -6,5 +6,5 @@ export function createListsAPI(config: KeystoneConfig, prismaClient: any) {
   const { getKeystone } = createSystem(initConfig(config));
```

Comment:
`createListsAPI` -> `createQueryAPI` or something + the thing we export from the `.keystone/api` thing should be renamed too, no?